release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,7 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * With this update, you can set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
+* With this update, you can see the labels regarding quota status in the *Topology* view, which includes an alert to exceeding quotas.
 
```

Comment:
@smunje there are two parts to this epic:
Resource quota alert label on the Topology and Add page:
1. The above alert takes the user to the search page with all rqs and if there is just one rq then it takes the user to the details page for the same.

For deployments, if there are any errors associated with resource quotas, a warning alert is shown in the side panel.
Also a yellow border is shown around the deployment nodes when resource quota exceeds.